geomstats/geometry/matrices.py

```diff
@@ -242,7 +247,7 @@ def is_diagonal(cls, mat, atol=gs.atol):
         return is_diagonal
 
     def random_point(self, n_samples=1, bound=1.):
-        """Sample from a uniform distribution.
+        """Sample from a uniform distribution in a cube.
```

Comment:
Should we rename them to `random_uniform` or `random_uniform_cube` then?

Reply:
No I think it is just useful to say how points are sample but let's keep `random_point` so that all manifolds have a random_point method